C7/Art/grid.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
apparently we should check all `*.import` files into source, but ***not*** Godot's `.import` folder